tvbextunicore/logger/logging.conf

```diff
@@ -0,0 +1,61 @@
+############################################
+## TVB-EXT-UNICORE - logging configuration.   ##
+############################################
+[loggers]
+keys=root, tvbextunicore, numba, matplotlib
+
+[handlers]
+keys=consoleHandler, fileHandler
+
+[formatters]
+keys=simpleFormatter
+
+[logger_root]
+level=INFO
+handlers=consoleHandler, fileHandler
+propagate=0
+
+############################################
+## tvbextunicore specific logging            ##
+############################################
+[logger_tvbextunicore]
+level=INFO
+handlers=consoleHandler, fileHandler
+qualname=tvbextunicore
+propagate=0
+
+[logger_numba]
+level=ERROR
+handlers=consoleHandler
+qualname=numba
+propagate=0
+
+[logger_matplotlib]
```

Comment:
I think matplotlib and numba are not 3rd party dependencies here. Are they?

Reply:
They are not, I removed them